### ISSUE TITLE
Missing CSS

### ISSUE THREAD

**Issue by nitinsinghit** [rweng/jquery-datatables-rails#160]
According to instructions I installed gem and ran "rails generate jquery:datatables:install". I see generated javascript and stylesheets files in respective folders. I am able to sort and search but there no sorting icons on columns and table css is missing. 

I am using rails4.1.6. IS there anything else I need to show correct css ?

Thanks

**Comment by rstudner** [comment]
I think i'm having the same issue.

I'm still on rails 3.1.x, but I see this in my console log:

GET http://localhost:3000/assets/images/sort_both.png 404 (Not Found)
jquery.dataTables.js?body=1:4246 
GET http://localhost:3000/assets/images/sort_desc.png 404 (Not Found)
jquery.js?body=1:5996 
GET http://localhost:3000/assets/images/sort_asc.png 404 (Not Found)

It is looking for the images in the wrong places it seems

**Comment by rstudner** [comment]
*= require dataTables/bootstrap/2/jquery.dataTables.bootstrap

That as well doesn't actually point at anything/do anything.

It appeals the datatables:install bootstrap2 doesn't really work either

**Comment by MasterSlowPoke** [comment]
You'll need to get the latest version from github, this fix hasn't been released to RubyGems yet

```ruby
gem 'jquery-datatables-rails', github: 'rweng/jquery-datatables-rails'
```

**Comment by tsai146** [comment]
confirmed @MasterSlowPoke 's solution works.

**Comment by rstudner** [comment]
Reeeeelease it to Rubygems ;)

**Comment by dongshengbc** [comment]
+1

**Comment by ricardodovalle** [comment]
I will release soon.
Thanks @MasterSlowPoke and @tsai146  :clap:

**Comment by ricardodovalle** [comment]
Thanks, I released a new gem version to rubygems (3.2.0).
If the error persist, please, reopen it.
